demos/signature/EVP_Signature_demo.c

```diff
@@ -0,0 +1,261 @@
+/*-
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/*
+ * Example of using EVP_MD*, EVP_DigestSign* and EVP_DigestVerify* methods
+ * to calculate and verify a signature of static buffers.
+ * Another way of signature is to use EVP_PKEY_sign(), it uses a digest
+ * of message, does not hash the data to be signed, and therefore is
+ * normally used to sign digests.
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include "EVP_Signature_demo.h"
+#include <openssl/decoder.h>
+
+/*
+ * This demonstration will show calculate and verify a signature of data using
+ * the soliloqy from Hamlet scene 1 act 3
+ */
+
+static const char *hamlet_1 =
+    "To be, or not to be, that is the question,\n"
+    "Whether tis nobler in the minde to suffer\n"
+    "The ?lings and arrowes of outragious fortune,\n"
```

Comment:
It's **ſlings** not ?lings

Reply:
Done in latest upload, thanks for your comments.